cmd/main.go

```diff
@@ -1,34 +1,41 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
+	"strings"
 	"weather"
 )
 
-type Response struct {
-	OneWord string  `json:"oneword"`
-	Celcius float64 `json:"celcius"`
+func LocationFromArgs(args []string) string {
+
+	var command strings.Builder
+
+	for i, v := range os.Args {
```

Comment:
We should be ranging over `args`, not `os.Args`, shouldn't we? The point being that when we call this function from a test, we can supply any arguments we want.